cmd/kops/create_cluster.go

```diff
@@ -359,6 +365,22 @@ func RunCreateCluster(f *util.Factory, cmd *cobra.Command, args []string, out io
 		}
 	}
 
+
+	// Network Topology
+	switch  c.Topology{
```

Comment:
Where do we add notes on what this actually means? n00b dev saying cool we have Topo but what the hell is Topo?